### ISSUE TITLE
Fix compute resource add image test case

### ISSUE THREAD

**Issue by san7ket** [SatelliteQE/robottelo#4983]
The image add test cases are failing due , to no operating system RedHat 7.3 present when the test are run. 
Add the os in the test , if not present.

-  tests.foreman.ui.test_computeresource_rhev.RhevComputeResourceTestCase.test_negative_add_image_rhev_with_invalid_name
- tests.foreman.ui.test_computeresource_rhev.RhevComputeResourceTestCase.test_positive_add_image_rhev_with_name
- tests.foreman.ui.test_computeresource_vmware.VmwareComputeResourceTestCase.test_negative_add_image_vmware_with_invalid_name

**Comment by rplevka** [comment]
this is because it searches for `{0}{1}.{2}.format(os.name, os.major, os.minor` however, that particular field uses `os.description` for listing/searching the os.
The result is:
```
'RedHat7.3' != 'RHEL Server 7.3'
```

**Comment by san7ket** [comment]
Looking on the passed test case for vmware[1] , it doesn't look like it , the OS is searched using the title parameter , which has to be created prior adding image to CR. 
In this test particularly the OS was created, maybe because of some other test and this test-case was passed.

- tests.foreman.ui.test_computeresource_vmware.VmwareComputeResourceTestCase.test_positive_add_image_vmware_with_name

```
2017-07-15 08:53:43 - robottelo.ui.browser - DEBUG - findElement: id: 28 {'using': 'xpath', 'value': u"//div[@id='select2-drop']//input"}
2017-07-15 08:53:45 - robottelo.ui.browser - DEBUG - findElement: id: 28 {'using': 'xpath', 'value': u"//div[@id='select2-drop']//input"}
2017-07-15 08:53:47 - robottelo.ui.browser - DEBUG - sendKeysToElement:  {'id': u'28', 'value': 'RedHat 7.3'}
2017-07-15 08:53:48 - robottelo.ui.base - DEBUG - Assigned value RedHat 7.3 to <|strategy=xpath|value=//div[@id='select2-drop']//input|>
2017-07-15 08:53:48 - robottelo.ui.browser - DEBUG - findElement: id: 29 {'using': 'xpath', 'value': u"//ul/li/div[normalize-space(.)='RedHat 7.3']"}
2017-07-15 08:53:50 - robottelo.ui.browser - DEBUG - clickElement:  {'id': u'29'}
2017-07-15 08:53:50 - robottelo.ui.base - DEBUG - Selected value RedHat 7.3 on <|strategy=xpath|value=//div[@id='s2id_image_operatingsystem_id']/a/span[contains(@class, 'arrow')]|>
2017-07-15 08:53:50 - robottelo.ui.base - DEBUG - Assigned value RedHat 7.3 to <|strategy=xpath|value=//div[@id='s2id_image_operatingsystem_id']/a/span[contains(@class, 'arrow')]|>
```

**Comment by rplevka** [comment]
@san7ket no, the RHEL OS is being automatically based on the OS the satellite runs on. This is automatic. The other way other OS-es are creted automatically is kicsktart repo sync.

One of the possible explanations is actually the one, where other test deleted this default OS

**Comment by san7ket** [comment]
The OS is searched on the basis of title field at-least for the Image creation wizard, I guess you might have RHEL Server 7.3 as a title of an os on that box , I tried it , but couldn't , it failed for 'RHEL Server 7.3'.
And yes you are right the Default OS must have been deleted by some test, I tried deleting/renaming the RedHat 7.3 , but after some time it was automatically re-created/populated. This explains why one test is passing and others fail.
